azure-quantum/azure/quantum/target/ionq.py

```diff
@@ -12,6 +12,24 @@
 from azure.quantum.workspace import Workspace
 from azure.quantum._client.models import CostEstimate, UsageEvent
 
+COST_1QUBIT_GATE_MAP = {
```

Comment:
nitpick: I would group these by target so every time we add a new target we add one structure instead of modifying 3.